packages/client/src/tx/types.ts

```diff
@@ -168,7 +188,8 @@ export type PaymentInfo = {
   partial_fee: bigint
 }
 
-export type Transaction<
+export type InnerTransaction<
+  Unsafe,
   Arg extends {} | undefined,
```

Comment:
Be careful, this type might leak out. I'm not quite familiar on how TS decided whether a type should be unwrapped or named.

Might be better to just duplicate the types, if this file gets too long maybe on another file (unsafeTypes.ts?)

Reply:
In my tests the type does not get leaked in the typed client. I want to add an `UnsafeTransaction` with the same structure to avoid leaking it to the Unsafe client.